data/test_response_selector_bot/tests/test_stories.yml

```diff
@@ -0,0 +1,30 @@
+version: "2.0"
+
+stories:
+- story: test 0 - both predictions should be the same in the test stories
+  steps:
+  - user: |
+      What is your name?
+    intent: chitchat/ask_name
+  - action: utter_chitchat/ask_name
+
+- story: test 1 - test retrieving full utter action when predicted wrongly
+  steps:
+  - user: |
+      What is your name?
+    intent: chitchat/ask_name
+  - action: utter_chitchat/ask_weather
+
+- story: test 2 - test that works well without retrieval intents 1
```

Comment:
shouldn't this one fail due to the wrong `utter_goodbye`?

Reply:
Yes, it does fail. And I'm checking `failed_test_stories.yml` to see if it failed correctly.

Reply:
should we fix the comment then?